wrapper/src/test/java/integration/util/ContainerHelper.java

```diff
@@ -49,7 +49,7 @@ public class ContainerHelper {
 
   private static final String MYSQL_CONTAINER_IMAGE_NAME = "mysql:latest";
   private static final String POSTGRES_CONTAINER_IMAGE_NAME = "postgres:latest";
-  private static final String MARIADB_CONTAINER_IMAGE_NAME = "mariadb:latest";
+  private static final String MARIADB_CONTAINER_IMAGE_NAME = "mariadb:10";
```

Comment:
Just curious, why are we changing this?

Reply:
The latest tag was updated 5 days ago to point to mariadb v11, and standard integration tests are failing.
Since we are close to the release we are changing the tag to v10 for now.